lib/gds_api/email_alert_api.rb

```diff
@@ -57,6 +57,13 @@ def send_alert(publication, headers = {})
     post_json("#{endpoint}/notifications", publication, headers)
   end
 
+  # Unpublishing alert
+  #
+  # @param message [Hash] base_path, content_id
```

Comment:
Maybe just say something about how it's the message the Publishing API sends about unpublished content? I'm not sure what "base_path, content_id" adds here.